File: src/commands/core/ServerSideCommand.java
package commands.core;

import lib.functions.CallbackFunction;
import organization.OrganizationManager;

public abstract class ServerSideCommand extends Command {
    protected OrganizationManager organizationManager;

    protected ServerSideCommand(CallbackFunction callbackFunction, OrganizationManager organizationManager) {
        super(callbackFunction);
        this.organizationManager = organizationManager;
    }
}

File: src/commands/client_side/core/ServerSideCommand.kt
package commands.client_side.core;

import OrganizationDatabase.OrganizationManagerInterface;

public abstract class ServerSideCommand extends Command {
    protected OrganizationManagerInterface organizationDatabase;

    protected ServerSideCommand(ClientCallbackFunction clientCallbackFunction, OrganizationManagerInterface organizationDatabase) {
        super(clientCallbackFunction);
        this.organizationDatabase = organizationDatabase;
    }
}
